introduction/architecture.md

```diff
@@ -111,6 +110,68 @@ while maintaining support for multiple indexes.
+The two roles that nodes can take on are:
+1) Access node
+2) Data node
+
+A front-end access node accepts all requests from the client, distributes the requests and queries
+appropriately to back-end data nodes, and aggregates the results received from the data nodes.
+Access nodes store cluster-wide information about the different data nodes as well as how
+chunks are distributed across those data nodes. Access nodes can also store non-distributed
+hypertables, as well as regular PostgreSQL tables.
+
+Back-end data nodes store data associated with distributed hypertables. Data nodes
+are not aware of cluster-wide distribution policies, and otherwise look just as if they were
+single TimescaleDB instances. However, certain actions are blocked on data nodes to protect
+from accidentally executing operations on data nodes that might endanger the data consistency
+of distributed hypertables.
+
+It is important to note that access nodes and data nodes both run TimescaleDB, and for all intents and
+purposes, look just like a single instance of TimescaleDB from an operational perspective.
+
+### Distributed Databases
+
+Distributed databases represent the highest level in the hierarchy used to define a multi-node
+TimescaleDB Cluster. Users can define access nodes and data nodes at the database level. This means
+that if you have multiple databases in a TimescaleDB Cluster, you could theoretically have
+different nodes play both the role of access node and data node for different databases.
+We do not typically suggest running in this manner. Instead, we recommend that you
+only have one access node that acts as a query routing layer between your application and
+all of your backend data nodes.
+
+### Distributed Hypertables
+
+As discussed above, distributed databases contain distributed hypertables. Distributed hypertables are essentially
+the same as regular hypertables, except that they have an additional layer of abstraction where
+access nodes must be able to logically manage distributed hypertables. Distributed hypertables
+are also composed of chunks.
+
+With distributed hypertables, we suggest partitioning the hypertables both by space and by time.
+If you only partition data by time, that chunk will have to fill up before the access node chooses another
+data node to store the next chunk. On the other hand, if you specify a space partition, the access node
+will distribute chunks across multiple data nodes based on the space partition so that multiple
+chunks are created for a given chunk time interval.
+
+We recommend that the number of space partitions are equal or greater than the number of data nodes associated with the distributed hypertable for optimal data distribution across data nodes. The system will automatically increase the number of space partitions, if necessary, when adding new data nodes. In case of multiple space partitions, only the first space partition will be used to determine how chunks are distributed across servers.
```

Comment:
Can we be stronger in recommendation?  Perhaps we recommend # space partitions *equal* to the # of data nodes?  (For example, if 5 servers and 6 partitions, I think this will actually work quite poorly.  Ultimately we might recommend some # space partitions that k*N for N data nodes, where k >= 1.  We just haven't yet performed more detailed benchmarking for k>1

Reply:
Changed to "equal or a multiple of" and also have additional reordering that I think makes this section much clearer